src/interfaces/.solhint.json

```diff
@@ -0,0 +1,11 @@
+{
+  "rules": {
+    "ordering": "off",
```

Comment:
Should we keep ordering on in interfaces?

Reply:
I like having a standard for interface ordering; following the layout from the Solidity docs convinces me—at the same time, I prefer putting default getters above other functional logic. I think this last preference conflicts with how `ordering` works.